.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,10 +26,10 @@ If applicable, add screenshots to help explain your problem.
 **Platform Information (please complete the following information):**
  - OS: [e.g. Windows 10 2004 / Ubuntu 20.10]
  - Docker: [Yes/No]
- - Mono or.NET Core Version: [e.g. Mono 5.8 or .Net Core 3.1.10] (found under System -> Status)
- - Browser and Version [e.g. chrome 86.0.4240.198] (Only needed for UI issues)
- - Radarr Version [e.g. 3.0.0.2956]
- - Radarr Branch [e.g. master]
+ - Mono or.NET Core Version (System -> Status): [e.g. Mono 5.8 or .Net Core 3.1.10] 
```

Comment:
Go ahead and change this to `Mono or .NET Version (System -> Status)` with eg. `[e.g. Mono 5.8 or .Net Core 3.1.10 or .NET 5.0.1] `